.pylintrc-mandatory

```diff
@@ -39,7 +39,6 @@ enable=anomalous-backslash-in-string,
     method-required-super,
     method-search,
     missing-import-error,
-    missing-manifest-dependency,
```

Comment:
Why removing this?

Reply:
The pylint was failing, even when adding the external-dependencies in manifest. I adopted the same solution as for `date_range`: https://github.com/OCA/server-ux/pull/99#issuecomment-543679437

Reply:
This workaround is not needed anymore since it's working fine after rebase.